test/mobile/test_lite_script_module.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import torch
+
+import io
+
+from torch.jit.mobile import *
```

Comment:
Please avoid `import *`, since that makes it hard to figure out where any given symbol comes from.